.circleci/config.yml

```
@@ -83,12 +83,32 @@ jobs:
           command: npm run prettier:check
       - docker/dockerlint
 
+  build-example:
+    docker:
+      - image: cimg/node:lts-browsers
+    steps:
+      - checkout
+      - node/install-packages
+      - run:
+          name: Build web version
+          command: |
+            npm start -- build --material=example
+            tar -cvzf dist.tar.gz dist
+      - store_artifacts:
+          path: dist.tar
```

https://app.circleci.com/pipelines/github/Zenika/sensei/477/workflows/f1dd620b-73dc-43ca-a03b-36df9fa01b1d/jobs/997/parallel-runs/0/steps/0-109 :
```
Uploading /home/circleci/project/dist.tar to dist.tar
  No artifact files found at /home/circleci/project/dist.tar
Total size uploaded: 0 B
```

```suggestion
          path: dist.tar.gz
```